src/app_service/service/message/async_tasks.nim

```diff
@@ -229,7 +229,7 @@ const asyncGetLinkPreviewDataTask: Task = proc(argEncoded: string) {.gcsafe, nim
 
     #2. Process whitelisted url
     #status deep links are handled internally
-    if domain == "status-im" or domain == "join.status.im":
+    if domain == "status-app" or domain == "join.status.im":
```

Comment:
Better use the defined constants?

Reply:
I think you only reviewed the first commit. I added a second one that uses the constants